packages/hooks/src/useBounding/__tests__/index.test.ts

```diff
@@ -0,0 +1,87 @@
+import { act, renderHook } from '@testing-library/react';
+import ResizeObserver from 'resize-observer-polyfill';
+import useBounding, { INIT_BOUNDING_RECT } from '../index';
+
+window.ResizeObserver = ResizeObserver;
+
+const windowListenerMock: any = {};
+const getBoundingClientRectMock = jest.spyOn(HTMLElement.prototype, 'getBoundingClientRect');
+jest
+  .spyOn(window, 'addEventListener')
+  .mockImplementation(
+    jest.fn((event: any, callback: any) => (windowListenerMock[event] = callback)),
+  );
+const INIT_VALUE = {
+  width: 200,
+  height: 200,
+  left: 100,
+  right: 500,
+  top: 100,
+  bottom: 500,
+  x: 100,
+  y: 100,
+} as DOMRect;
```

Comment:
what about moving these into `describe(` block, then restoring spy mocks at `afterAll(`?

Reply:
yeah, it's a good idea

Reply:
done